### ISSUE TITLE
Android Studio updated. Gradle 4 removed compile

### ISSUE THREAD

**Issue by carlosbrolo** [agrosner/DBFlow#1599]
DBFlow Version: 4.2.4

Bug or Feature Request: Cant compile anymore.  Android Studio is using Gradle 4 now. They removed compile from build process.

Description:

**Comment by carlosbrolo** [comment]
The only way I made it to work:
Searched for compiled jars on my local gradle folder
and change the following lines on my gradle config:

```
implementation files('libs/dbflow-4.2.4.jar')
implementation files('libs/dbflow-core-4.2.4.jar')
annotationProcessor "com.github.Raizlabs.DBFlow:dbflow-processor:4.2.4"


```

**Comment by agrosner** [comment]
this will be fixed in 5.0

**Comment by agrosner** [comment]
in master.
